src/tvheadend.h

```diff
@@ -334,10 +334,16 @@ void tvh_qsort_r(void *base, size_t nmemb, size_t size, int (*compar)(const void
 # endif /* ULONG_MAX */
 #endif /* __WORDSIZE */
 
-#if __WORDSIZE == 32 && defined(PLATFORM_FREEBSD)
-#define PRItime_t       "d"
```

Comment:
I think you based your MR not on master. And the problem is a bit more difficult then this, right now, we still see breakage on at least debian on raspberry pi.

Feel free to so share thoughts and insights in #1548